### ISSUE TITLE
Zauberseite: Repräsentation anzeigen, Zauber mehrmals erlernen

### ISSUE THREAD

**Issue by Rondragos** [kreuvf/roll20-character-sheets#87]
Für die Zauberseite wäre es praktisch, Zauber mehrmals aktivieren zu können, zum Beispiel, wenn man sie in unterschiedlichen Repräsentationen lernen kann oder bei Zaubern wie dem Arcanovi oder dem Adlerschwinge, von denen man durchaus mehrere Varianten lernen kann. 
Überhaupt wäre es gut, die Zauber mit Repräsentation eintragen zu können.

**Comment by kreuvf** [comment]
Ich stimme dir in allen Punkten zu und das Zaubertab insgesamt (also auch mit Ritualen und Sonderfertigkeiten) hat noch viel Arbeit vor sich.
